tutorials/cloud-iot-logging/functions/src/index.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2018 Google LLC
```

Comment:
* Copyright 2018, Google, Inc.

Reply:
good catch - this was total copy paste from elsewhere in the repo